### ISSUE TITLE
Странный эффект - из "Центра нотификаций" не работает переход по url

### ISSUE THREAD

**Issue by freerider7777** [peter-gribanov/serviceworker#7]
Добрый день! 
Когда открыта страница с демо https://peter-gribanov.github.io/serviceworker/
Шлем сообщение (Send) - сообщение приходит, всё ок. Кликаем по нотификейшену пока он показывается - переходим по ссылке. Однако, если послать и подождать пока оно "спрячется" в список в "Центре нотификаций" и кликать там - переход по ссылке не происходит почему-то...
Windows 10 Chrome Version 83.0.4103.116 (Official Build) (64-bit)
и в Опере тоже самое кстати

**Comment by peter-gribanov** [comment]
Вероятно это проблема конкретно Windows.
Переход по ссылке выполняется по клику на уведомлении [`Notification.onclick`](https://developer.mozilla.org/ru/docs/Web/API/Notification/onclick), и вероятно это событие не вызывается из "Центра уведомлений" Windows.

**Comment by freerider7777** [comment]
воспроизвелось? что интересно, оно в принципе работает - есть экзампл ("welcome message" можно жать - придет notification, который срабатывает из Центра уведомлений)
https://www.webpushnotifications.com

**Comment by peter-gribanov** [comment]
>воспроизвелось?

Нет. У меня нет под рукой нормальной Windows, а в той, что есть, push уведомления не попадают в "Центра уведомлений" Windows.
